### ISSUE TITLE
Edited Video Section (added border to videos and labels to videos)

### ISSUE THREAD

**Issue by MadalapureVaishnavi** [TechNodes2-0/ElectiveHub#269]
### Is your feature request related to a problem? Please describe.

![Screenshot 2024-06-03 142448](https://github.com/TechNodes2-0/ElectiveHub/assets/89457366/fa52e73a-29b1-4ff9-a8f4-17b94f9f553d)

assign this task to me

### Checklist

- [X] I have checked the [existing issues](https://github.com/TechNodes2-0/ElectiveHub/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/TechNodes2-0/ElectiveHub/blob/main/CONTRIBUTING.md)
- [X] I want to work on this issue. (optional)

**Comment by Yash636261** [comment]
can you do shadow instead of giving border on hover? will assign to you after confirmation.

**Comment by MadalapureVaishnavi** [comment]
Yes, I've implemented the shadow effect instead of giving a border on hover. Here's a screenshot showcasing the updated UI.
![Screenshot 2024-06-08 235710](https://github.com/TechNodes2-0/ElectiveHub/assets/89457366/88a77496-ba7c-4476-ab36-5937a20b4bcb)

**Comment by MadalapureVaishnavi** [comment]
Please assign me this issue
